### DISCUSSION TITLE
在以行写入文件时，最后一行数据没有加入换行符 ，导致linux统计数量不对。

### DISCUSSION THREAD

**Discussion by esunshine** [looly/hutool#2954] [General]
测试代码：
`String srcFile = "D:\\test.txt";
List<String> list = FileUtil.readLines(srcFile,"UTF-8");
list .add("test");FileUtil.writeLines(list,srcFile,"UTF-8");
FileUtil.writeLines(list,srcFile,"UTF-8");
`

在linux下使用
wc -l test.txt 最后一行无法统计。

**Comment by looly** [comment]
可以手动补一行""
